### ISSUE TITLE
which objective function should i use

### ISSUE THREAD

**Issue by sixmilesroad** [UKPLab/sentence-transformers#836]
in the paper you order three objective functions ,1：Classification Objective Function  2：Regression Objective Function 3：Triplet Objective Function。 now the data format is   sentence1   sentence2   similarity score。which Objective Function should i use 。in the paper you test SBERT embedding on SentEval，in this test which objective function do you use? if you use Classification Objective Function does it mean that if use Regression Objective Function the SBERT embedding can not perform good on SentEval?

**Comment by nreimers** [comment]
Have a look at our documentation:
https://www.sbert.net/docs/package_reference/losses.html

For your task the cosine similarity loss sounds right.

The loss function is completely independent of SentEval

**Comment by sixmilesroad** [comment]
if the data format i have is sent1 sent2 lable(lable can be 0 or 1) i want know SoftmaxLossand ContrastiveLoss which should i try

**Comment by nreimers** [comment]
ContrastiveLoss  seems more intuitive

Check this example:
https://github.com/UKPLab/sentence-transformers/tree/master/examples/training/quora_duplicate_questions

**Comment by sixmilesroad** [comment]
i want know how to choose lossfunction. depends on the data format? or depends on the generated embedding use which task

**Comment by sixmilesroad** [comment]
in your paper in SentEval part,for MRPC dataset which lossFunction do you use?
